DeAceroDLabs/marketplace-frontend
# Estado de loading para obtener ofertas

## What is done

[[DMP-724] Estado de loading para obtener ofertas](https://deacero.atlassian.net/browse/DMP-724)

## How is done

1. Updated Offers to call upon LoadingOffers
2. Created LoadingOffers component

## How to test

1. Run the project via `npm start`
3. Head over to shopping cart
4. Click on the second step on stepper
5. LoadingOffers should pop up and a second later the info should load

## Evidence
![image](https://user-images.githubusercontent.com/116199447/208088974-9e18d91c-2979-49ae-9252-2c2b453c1106.png)
